frontend/src/Friends.tsx

```diff
@@ -17,22 +18,12 @@ const Friends = (props: Props) => {
           if (!friend.card) {
             return null;
           }
-          if (friend.skip === 0) {
-            return (
-              <p key={idx}>
-                The next person to play <InlineCard card={friend.card} /> is a
-                friend
-              </p>
-            );
-          } else {
-            return (
-              <p key={idx}>
-                {friend.skip} <InlineCard card={friend.card} /> can be played
-                before the next person to play <InlineCard card={friend.card} />{' '}
-                is a friend
-              </p>
-            );
-          }
+          return (
+            <p key={idx}>
+              The person to play the {nth(friend.initial_skip + 1)} <InlineCard card={friend.card} /> is a
+              friend. {(props.showPlayed) ? friend.initial_skip - friend.skip + ' played in previous hands' : ''}
```

Comment:
perhaps, "played in previous tricks"?

Reply:
Sure. I will make it consistent with the current terms